### ISSUE TITLE
What portion of OpenWebText2 did you use as validation set for pre-training?

### ISSUE THREAD

**Issue by ethancaballero** [christinakim/scaling-laws-for-language-transfer#6]
What portion of OpenWebText2 did you use as validation set for pre-training?
